Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM bigcommerce/ruby:2.0.0-p576
+FROM hub.services.bigcommerceapp.com/bigcommerce/ruby:2.0.0-p576
```

Comment:
we should probably start from a public image... what do you @bigcommerce/ruby-sf  think?